moped-editor/src/views/projects/projectView/NoteInputQuill.js

```diff
@@ -9,8 +9,8 @@ import {
   RadioGroup,
   FormControlLabel,
 } from "@mui/material";
-import ReactQuill from "react-quill";
-import "react-quill/dist/quill.snow.css";
+import ReactQuill from "react-quill-new";
+import "react-quill-new/dist/quill.snow.css";
```

Comment:
I see one compile warning related to this theme import, but it won't be there long so I'm not concerned about it. 👍